index.html

```diff
@@ -185,7 +184,7 @@ <h3>New Normative Specifications</h3>
       </p>
     </dd>
 
-    <dt id="webvtt" class="spec">
+    <dt id="webvtt" class="spec"><span class="todo">This is listed in previouysly published normative specifications</span>
```

Comment:
I don't understand - why not move it then, rather than adding the todo?

Reply:
wanted to ask whether to remove or not. removed at ae7b999